documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc

```diff
@@ -242,4 +242,35 @@ Specifically, ensure that the following properties have the same value across al
 * `topic.filter.class`
 
 For example, the value for `replication.policy.class` must be the same for the source, checkpoint, and heartbeat connectors.
-Mismatched or missing settings cause issues with data replication or offset syncing, so it's essential to keep all relevant connectors configured with the same settings.
\ No newline at end of file
+Mismatched or missing settings cause issues with data replication or offset syncing, so it's essential to keep all relevant connectors configured with the same settings.
+
+== Listing the offsets of MirrorMaker 2 connectors
+
+To list the offset positions of the internal MirrorMaker 2 connectors, use the same configuration that's used to manage Kafka Connect connectors.
+
```

Comment:
There is an extra difference for MirrorMaker. When you annotate the resource you have to set two annotations: `strimzi.io/connector-offsets=list` and `strimzi.io/mirrormaker-connector=[CONNECTOR_NAME]` (e.g. `strimzi.io/mirrormaker-connector=my-cluster-source->my-cluster-target.MirrorSourceConnector`)